bitrise.yml

```diff
@@ -0,0 +1,32 @@
+format_version: "11"
+default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
+
+workflows:
+  check:
+    steps:
+    - git::https://github.com/bitrise-steplib/steps-check.git:
+        inputs:
+        - skip_go_checks: "yes"
+        - workflow: lint
+
+  e2e:
+    steps:
+    - git::https://github.com/bitrise-steplib/steps-check.git:
+        inputs:
+        - workflow: e2e
+
+  generate_readme:
+    steps:
+    - git::https://github.com/bitrise-steplib/steps-readme-generator.git: { }
+
+  run_step:
```

Comment:
What is this workflow's purpose? It doesn't seem to do much

Reply:
By convention there is a sample workflow for local testing in the steps (called `sample`), however this is not a very good one, so removed.